src/Symfony/Component/Form/Extension/Csrf/Type/FormTypeCsrfExtension.php

```diff
@@ -54,8 +54,7 @@ public function __construct(CsrfTokenManagerInterface $defaultTokenManager, bool
     /**
      * Adds a CSRF field to the form when the CSRF protection is enabled.
      *
-     * @param FormBuilderInterface $builder The form builder
-     * @param array                $options The options
+     * @param array $options The options
```

Comment:
there are alot of useless `array $options` phpdocs as we already have the array typehint

Reply:
removed while merging